nio/client/async_client.py

```diff
@@ -356,13 +360,16 @@ def __init__(
         config: Optional[AsyncClientConfig] = None,
         ssl: Optional[bool] = None,
         proxy: Optional[str] = None,
+        presence: Optional[str] = None,
```

Comment:
It doesn't make sense to take a `presence` argument on client creation, the attribute is only relevant (and set) when syncing or using the related functions.

Reply:
Fixed